pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannel.java

```diff
@@ -51,7 +51,7 @@ public interface ServiceUnitStateChannel extends Closeable {
      *
      * ServiceUnitStateChannel elects the separate leader as the owner broker of the system topic in this channel.
      */
-    CompletableFuture<Optional<String>> getChannelOwnerAsync();
+    CompletableFuture<String> getChannelOwnerAsync();
```

Comment:
We should keep using the `Optional` here. I see the comments [here](https://github.com/apache/pulsar/pull/19102/files#r1059201582). But the root cause is that you should not throw an unchecked exception when leader is empty. It's an expected error and we should pass the result (leader is empty, i.e. an empty `Optional` is returned) to the caller side.

Reply:
Good point. Changed back.